apps/projection/src/app/ui/card/card.component.ts

```diff
@@ -1,34 +1,42 @@
 import { NgFor, NgIf } from '@angular/common';
-import { Component, Input } from '@angular/core';
-import { randStudent, randTeacher } from '../../data-access/fake-http.service';
-import { StudentStore } from '../../data-access/student.store';
-import { TeacherStore } from '../../data-access/teacher.store';
-import { CardType } from '../../model/card.model';
+import {
+  Component,
+  EventEmitter,
+  Input,
+  Output,
+  ViewEncapsulation,
+} from '@angular/core';
+import { Item } from '../list-item/item';
 import { ListItemComponent } from '../list-item/list-item.component';
 
 @Component({
   selector: 'app-card',
-  templateUrl: './card.component.html',
+  template: `
+    <div class="border-2 border-black rounded-md p-4 w-fit flex flex-col gap-3">
+      <img [src]="image" width="200px" />
+
+      <section>
+        <app-list-item
+          *ngFor="let item of list"
+          [item]="item"
+          (delete)="delete.emit(item.id)">
+        </app-list-item>
+      </section>
+
+      <button
+        class="border border-blue-500 bg-blue-300 p-2 rounded-sm"
+        (click)="add.emit()">
+        Add
+      </button>
+    </div>
+  `,
```

Comment:
This is working, but I think I should have ask for a more complex card to add. 

What if I want to add a text instead of a Title for my header ? I want to have a more highly customizable component. 
You should look at ng-content. 

Same for my list. What if I want to add a checkbox or another button ? In this case, I can only pass a name and id. 
You should look at ngTemplateOutlet. 

And having outputs going multiple components up, is not a very nice design, IMO.

Reply:
Ok so you want me to play with ng-content and ng-content-outlet .. this is maybe why the exercise is named Projection :D :D

Reply:
Exactly 🤣🤣🤣